src/watchlists/index.jsx

```diff
@@ -1,9 +1,45 @@
-import React from 'react';
-import { View, Text } from 'react-native';
-import commonStyles from '../common/styles';
-
-export default () => (
-  <View style={commonStyles.container}>
-    <Text>This is the watchlists screen</Text>
-  </View>
-);
+import React, { useState } from 'react';
+import {
+  ScrollView,
+  TouchableHighlight,
+  Image,
+  View,
+} from 'react-native';
+import { useSelector } from 'react-redux';
+import WatchList from './WatchList';
+import Popup from './Popup';
+import addIcon from '../../assets/icons/watchList/add.png';
+import styles from './styles';
+import commonStyle from '../common/styles';
+
+const navigationOptions = {
+  title: 'WATCHLISTS',
```

Comment:
does it need to be uppercase? If it is on the design then we can do it with pure css i think